STEP_Project/tests/popupTest.js

```diff
@@ -0,0 +1,74 @@
+let ar = document.getElementById("addRemove");
+let s = document.getElementById("select");
+/**
+ * testing function findPagecase
+ */
+describe('Testing pageType that determines enabling/disabling of buttons depending on URL', ()=>{
```

Comment:
Add a test case when u/[0-9] is not present.

Reply:
Added in the next PR